nncf/common/graph/layer_attributes.py

```diff
@@ -108,14 +108,19 @@ class LinearLayerAttributes(WeightedLayerAttributes):
     def __init__(self,
                  weight_requires_grad: bool,
                  in_features: int,
-                 out_features: int):
+                 out_features: int,
+                 bias: bool):
         super().__init__(weight_requires_grad)
         self.in_features = in_features
         self.out_features = out_features
+        self.bias = bias
 
     def get_weight_shape(self) -> List[int]:
         return [self.out_features, self.in_features]
 
+    def get_bias_shape(self) -> int:
+        return self.out_features if self.bias is True else 0
+
```

Comment:
add test for this method at relevant unit test

Reply:
This may affect the tensorflow side at https://github.com/vuiseng9/nncf/pull/10/commits/f3a5c231c1ff136778c4379467326287fcdd0a9a .
Added test at https://github.com/vuiseng9/nncf/pull/10/commits/ad49e4c33a682b3bffe2be12bcb1fe87b053b4f1 and https://github.com/vuiseng9/nncf/pull/10/commits/7b573c896176a0a6ec66c839258c5fed3faaa9ac